graph/options.go

```diff
@@ -33,11 +33,19 @@ const (
 	BoxByNamespace            string = "namespace"
 	BoxByNone                 string = "none"
 	NamespaceIstio            string = "istio-system"
+	RateNone                  string = "none"
+	RateReceived              string = "received" // tcp bytes received, grpc response messages, etc
+	RateRequests              string = "requests" // request count
+	RateSent                  string = "sent"     // tcp bytes sent, grpc request messages, etc
+	RateTotal                 string = "total"    // Sent+Received
 	defaultBoxBy              string = BoxByNone
 	defaultDuration           string = "10m"
 	defaultGraphType          string = GraphTypeWorkload
 	defaultIncludeIdleEdges   bool   = false
 	defaultInjectServiceNodes bool   = false
+	defaultRateHttp           string = RateRequests
+	defaultRateGrpc           string = RateRequests
```

Comment:
RED ALERT!!!! What blasphemous code doth my eyes witness?

```suggestion
	defaultRateGrpc           string = RateRequests
	defaultRateHttp           string = RateRequests
```

Reply:
O M G, will fix.